css/sheet.css

```diff
@@ -4,20 +4,77 @@ body{
 }
 header{
     margin:0;
-    padding:0;
+   padding:0;
    text-align: center;
    width:100%;
 }
-img{
+section{
+    background: #2E2E2E;
```

Comment:
@Ntwali-Josue Remove background colour in section. Create a bg-gray class for this background